packages/logging_cloudwatch/aws_logging_cloudwatch/lib/src/cloudwatch_logger_plugin.dart

```diff
@@ -128,11 +133,14 @@ class CloudWatchLoggerPlugin extends AWSLoggerPlugin
   Future<void> _startSyncingIfNotInProgress() async {
     Future<void> startSyncing() async {
       final batchStream = _getLogBatchesToSync();
+
```

Comment:
Why did you add these spaces in?  While some might make sense, having a new line in between a comment and the line it's commenting seems a bit strange: 

// set logs to end before the index.

logs.removeRange(tooNewStartIndex, events.length);

Reply:
Have you run `dart analyze` to remove any warnings beforehand as well?  Perhaps it's not necessary since this is a feature branch, but might be good to start running those to get ahead of the warnings (if there are any)